static/js/en.js

```diff
@@ -0,0 +1,134 @@
+            'replace_main': 'Adding a main.py file will replace the code in the editor!',
+            'replace_file': 'Do you want to replace the "{{file_name}}" file?',
+            'replace_module': 'Do you want to replace the "{{module_name}}" module?'
+          },
+          'code': {
+            'start': "# Add your Python code here. E.g.\n" +
+                     "from microbit import *\n\n\n" +
+                     "while True:\n" +
+                     "    display.scroll('Hello, World!')\n" +
+                     "    display.show(Image.HEART)\n" +
+                     "    sleep(2000)\n"
+          },
+          'drop': {
+            'python': 'Extracted from a Python file',
+            'hex': 'Extracted from a hex file'
+          },
+          'messagebar': {
+            'title': 'This editor is out of date!',
+            'link': 'Click here',
+            'instructions': 'to open the latest version.',
+            'close': 'Hide'
+          },
+          'load': {
+            'save-hex' : 'Download Hex',
+            'show-files' : 'Show Files',
+            'load-title': 'Load/Save',
+            'program-title': function() {
+              return $("#script-name").val();
+            },
+            'instructions': 'Drag and drop a .hex or .py file in here to open it.',
+            'submit': 'Load',
+            'toggle-file': 'Or browse for a file',
+            'fs-add-file': 'Add file'
+          },
+          'buttons':{
+            'command-download':{
+              'title': 'Download a hex file to flash onto your micro:bit',
+              'label': 'Download'
+            },
+            'command-files':{
+              'title':'Load/Save files',
+              'label':'Load/Save'
+            },
+            'command-connect':{
+              'title':'Connect to your micro:bit',
+              'label':'Connect'
+            },
+            'command-blockly':{
+              'title':'Click to create code with blockly',
+              'label':'Blockly'
+            },
+            'command-snippet':{
+              'title':'Click to select a snippet (code shortcut)',
+              'label':'Snippets'
+            },
+            'command-help':{
+              'title':'Discover helpful resources',
+              'label':'Help'
+            },
+            'command-language':{
+              'title':'Select a language',
+              'label':'Language'
+            },
+            'command-share':{
+              'title':'Create a link to share your script',
+              'label':'Share'
+            },
+            'command-serial':{
+              'title':'Connect to your micro:bit via serial',
+              'label':'Open Serial'
+            }
+          }
+        },
+        'microPythonApi': microPythonApi,
+        'flags': {
+          'blocks': false,
+          'snippets': true,
+          'experimental': false,
+          'share': false,
+          'options': true
+        },
+    };
```

Comment:
We need to remove the `microPythonApi` and `flags` from these files.

Reply:
These files should no longer be there (been replaced by files inside /lang/) so en.js and uppercase.js inside static/js/ will be deleted.